src/content/1.7/contribute/contribution-guidelines/_index.md

```diff
@@ -102,75 +124,127 @@ Remember: **smaller changes are easier to review, easier to test and easier to m
 
-Once PrestaShop releases a new minor ("dot-zero") version, it won't release new patch versions for previous minor versions – with the exception of rare cases, for example if a security bug is found just before or after a minor release is published.
+Type | Scope
+-----|------
+bug fix | The changes fix a bug.
+improvement | The changes improve an already existing feature (eg. cosmetic or UX changes, performance improvement, etc). 
+new feature | The changes introduce a functionality that didn't exist before (eg. add a button, a new page, a new block...)
+refacto | The changes only refactor code, without changing any of its side effects.
 
-This means that only the latest minor version patch branch is supported.
+The category is the portion of the project to which your changes apply to. We use this code alongside with the type to construct the [changelog](https://github.com/PrestaShop/PrestaShop/blob/develop/docs/CHANGELOG.txt) by grouping changes in different sections. Choose the code that most closely describes your change:
 
-For example, the `1.7.4.x` branch is supported until the release of version 1.7.5.0. After that, the only supported version branch will be `1.7.5.x`, and so on.
+Code | Scope
+-----|------
+FO | The changes impact the Front Office
+BO | The changes impact the Back Office
+IN | The changes impact the Installer
+WS | The changes impact the Web Services
+CO | The changes impact the Core (non-visible functionality)
+LO | The changes impact localization functionality
+TE | The changes impact automated tests
+ME | The changes only import a git branch into another (eg. merge maintenance branch into develop)
+PM | The changes are related to project management (eg. edit Github pull request form)
 
-If you find a bug on an unsupported version, make sure that bug is still present in the latest version. If the bug is still present, please submit a PR on `develop`.
+{{% notice note %}}
+Remember, this is only needed for the Pull Request form, not for your commit messages.
+{{% /notice %}} 
 
-**PRs on unsupported versions will be closed.**
+#### BC breaks
 
-{{% notice tip %}}
-When in doubt, use the develop branch. We will ask you to rebase on the correct branch if necessary.
+It is very important to note if your change introduces backwards incompatible changes ("backward compatiblity breaks").
+
+Here are some examples of changes that can be considered breaking changes:
+
+* Change any public method signature in any class (rename, change its return type or structure, remove a parameter, or change its type).
+* Change any public property in any class.
+* Rename, move, or delete any class or class namespace.
+* Add new requirements (like dropping support for old versions of PHP or browsers, requiring new server-side libraries, etc.)
+* Replace any subsystem (like updating libraries to new major versions or replacing a library with another).
```

Comment:
Possible improvement: add examples of BC breaks for markup (e.g. modify html markup hierarchy, remove/rename id/classes, modify existing css class ...)